source/core/src/main/com/csse3200/game/areas/ForestGameArea.java

```diff
@@ -381,6 +374,26 @@ private Entity spawnPlayer(GridPoint2 position) {
     return newPlayer;
   }
 
+  // commented 383 - 386 out as there was a missing arg?
+//  private void spawnDemonBoss() {
+//    Entity demon = MobBossFactory.createDemonBoss();
+//    spawnEntityAt(demon, new GridPoint2(19, 5), true, false);
+//  }
+
+  private void spawnPatrick() {
```

Comment:
I think this mob spawn has been removed, as it's now covered in the switch statement of spawnMob. @samsully can you confirm?

Reply:
Yeah ForestGameArea is constantly changing, we may need to deal with this in the next sprint. We haven't modified any of the mob spawning in this file, but have removed any hardcoded tower spawning as it's not necessary now.